app/views/registers/show.html.haml

```diff
@@ -89,8 +89,8 @@
                 = search_field_tag 'q', nil, class: 'search-input', placeholder: 'Search', value: params[:q]
                 = submit_tag 'Search', name: nil, class: 'search-submit'
             .column-full
-              %details{role: "group", open: (params[:status].present? && @records.present? ? 'open' : nil)}
```

Comment:
I wonder if we should check if `params[:q]` has a value as opening the filter does seem pointless if we are showing the same data anyway.

Reply:
The case here is when you have selected a `params[:status]` e.g. Archived but there are no results. So in that case it is useful to see the filter even if there is no `params[:q]

Reply:
Ah okay. I tested this by searching and empty query which just reloaded the page and opened the filter section which seemed off but I see what you're saying 👌

Reply:
Erm, this was the behaviour already in `beta-assessment`. So is unrelated to my PR. It's becuase the status param is set by default in the form even though it is not visible.